### PR TITLE
Add omi-no2-nasa

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
*Description of changes:*

Add documentation for OMI NO2 data

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: eb5872307db391e23b130d507beb992ff2e43193
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=b1c5e8f7-8779-47f3-a968-e5237810d9c5%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: e3da697897f622f910da6ff30a8f41a32e47d799
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=9a52ffac-cb7d-4fce-b73c-f4bc4fad66e3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: ecfdefd6be75012e4e8923e721850a9a168d0c61
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=c516d26d-e355-42ab-a678-7105b2059296%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: ff27f2d872c9524656929fa829670694385db59d
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=8eaf2821-7d55-4b8b-b0d9-232a1691d73c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/omi-no2-nasa.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Name: OMI/Aura NO2 Cloud-Screened Tropospheric Column Global Gridded 0.25 degree x 0.25 degree in Cloud-Optimized GeoTiff
```
Comment (jflasher) [inline datasets/omi-no2-nasa.yaml:1]:
```suggestion
Name: Ozone Monitoring Instrument (OMI) / Aura NO2 Tropospheric Column Density
```

File: datasets/omi-no2-nasa.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Description: |
+  NO2 tropospheric column density, screened for CloudFraction < 30% global daily  composite at 0.25 degree resolution for the temporal range of 2004 to May 2020.
+  Original archive data in HDF5 has been processed into a Cloud-Optimized GeoTiff (COG) format. Quality Assurance - This data has been validated by the NASA Science Team at Goddard Space Flight Center.
+  Cautionary Note: https://airquality.gsfc.nasa.gov/caution-interpretation.
+  File Listing: https://omi-no2-nasa.s3-us-west-2.amazonaws.com/.
```
Comment (jflasher) [inline datasets/omi-no2-nasa.yaml:6]: I think we can remove this as this is not useful when viewed via http?

File: datasets/omi-no2-nasa.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Name: OMI/Aura NO2 Cloud-Screened Tropospheric Column Global Gridded 0.25 degree x 0.25 degree in Cloud-Optimized GeoTiff
+Description: |
+  NO2 tropospheric column density, screened for CloudFraction < 30% global daily  composite at 0.25 degree resolution for the temporal range of 2004 to May 2020.
+  Original archive data in HDF5 has been processed into a Cloud-Optimized GeoTiff (COG) format. Quality Assurance - This data has been validated by the NASA Science Team at Goddard Space Flight Center.
```
Comment (jflasher) [inline datasets/omi-no2-nasa.yaml:4]:
```suggestion
  Original archive data in HDF5 has been processed into a [Cloud-Optimized GeoTiff (COG)](https://www.cogeo.org/) format. Quality Assurance - This data has been validated by the NASA Science Team at Goddard Space Flight Center.
```

File: datasets/omi-no2-nasa.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  - COVID-19
+  - environmental  
+License: There are no restrictions on the use of these data.
+Resources:
+  - Description: S3 Bucket for OMI NO2 in Cloud-Optimized GeoTiff format
+    ARN: arn:aws:s3:::omi-no2-nasa
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tools & Applications:
```
Comment (jflasher) [inline datasets/omi-no2-nasa.yaml:25]: Can you add a link in here of the viewer actually displaying a COG image?

**Pull request merged**